region/crateria/east/Crab Maze.json

```diff
@@ -107,6 +107,7 @@
         {"resetRoom": {
           "nodes": [1]
         }},
+        "Morph",
         {"refill": ["Energy", "PowerBomb"]}
```

Comment:
I feel like this farm might be slow with just morph and using the right door. Maybe Energy should be limited to like 1-200 and PBs to 5 ish or something